Library/Homebrew/shims/linux/super/make

```diff
@@ -21,6 +10,16 @@ pathremove () {
         export $PATHVARIABLE="$NEWPATH"
 }
 
-pathremove "$HOMEBREW_LIBRARY/Homebrew/shims/linux/super"
+if [[ -n "$HOMEBREW_MAKE" && "$HOMEBREW_MAKE" != "make" ]]
+then
+  export MAKE="$HOMEBREW_MAKE"
+else
+  SAVED_PATH="$PATH"
+  pathremove "$HOMEBREW_LIBRARY/Homebrew/shims/linux/super"
+  export MAKE="$(which make)"
```

Comment:
Please change `which` to `command -v` as suggested by @iMichka.

Reply:
I'm happy with `command -v` (rather than `type -pP`).

Reply:
See my comment above https://github.com/Homebrew/brew/pull/4570#issuecomment-409275957

Reply:
I don't believe aliases affect non-interactive shells. I'm okay with using the more typical idiom `command -v`.

Reply:
true! forgot we're doing this in a script